content/docs/concepts/security/index.md

```diff
@@ -628,9 +628,8 @@ Each rule has the following standard fields:
 - **`services`**: A list of service names. You can set the value to `*` to
   include all services in the specified namespace.
 
-- **`methods`**: A list of HTTP method names, for permissions on gRPC requests,
-  the HTTP verb is always `POST`. You can set the value to `*` to include all
-  HTTP methods.
+- **`methods`**: A list of HTTP methods. You can set the value to `*` to include all
+  HTTP methods. This field should not be used for TCP and gRPC services.
```

Comment:
NIT: "This field should not be set for TCP and gRPC services. "

used => set.

Reply:
Done. Thanks!